### PR TITLE
Include pylint in `tests_require`

### PR DESCRIPTION
**Pull request by andy-faunadb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ examples.
 
   virtualenv venv
   source venv/bin/activate
-  pip install .
+  pip install -e .
+  pip install nose pylint pynt sphinx sphinx_rtd_theme testfixtures
```
Comment (ashfire908) [inline README.md]: Maybe we could move what's needed for testing and what's needed for docs into the two sections below? None of this is required to simply use it.

Comment (andy-faunadb) [inline README.md]: I'll remove this and modify the python3 branch instead.

**Comment by ashfire908** [comment]
Update commit message to reflect changes?

**Comment by ashfire908** [comment]
:sheep:

**Pull request merged**